spec/pageflow/policies/account_policy_spec.rb

```diff
@@ -0,0 +1,20 @@
+require 'spec_helper'
+
+module Pageflow
+  module Policies
+    describe AccountPolicy do
+      terms = {class: AccountPolicy,
+               subject: FactoryGirl.create(:account),
```

Comment:
Creating models outside of test runs does not work correctly with database resets in before hooks.